src/web/handlers/status_list/error.rs

```diff
@@ -45,6 +47,7 @@ impl IntoResponse for StatusListError {
             DecompressionError(_) => StatusCode::BAD_REQUEST,
             CompressionError(_) => StatusCode::INTERNAL_SERVER_ERROR,
             StatusListAlreadyExists => StatusCode::CONFLICT,
+            IssuerMismatch => StatusCode::BAD_REQUEST,
```

Comment:
403 is more appropriate in this situation.
```suggestion
            IssuerMismatch => StatusCode::FORBIDDEN,
```